packages/guides/src/Compiler/NodeTransformers/MenuNodeTransformer.php

```diff
@@ -38,6 +38,10 @@ public function enterNode(Node $node, DocumentNode $documentNode): Node
             }
 
             foreach ($this->buildFromDocumentEntry($metaEntry, 1, $node) as $entry) {
+                if ($entry->getUrl() === $documentNode->getFilePath()) {
```

Comment:
should this not be view/template behavior? all nodes will become active like this?

Reply:
This sets a flag on a documentEntry if (and only if) the document entry equals the current document. The view can then decide on if and how to display this information

Reply:
Oh, I think I was wrong here, the menu nodes are not shared between pages. This only applies to metas.